build/meson/lib/meson.build

```diff
@@ -77,8 +77,12 @@ endif
 libzstd_c_args = []
 if cc_id == compiler_msvc
   if default_library_type != 'static'
-    libzstd_sources += [windows_mod.compile_resources(
-      join_paths(zstd_rootdir, 'build/VS2010/libzstd-dll/libzstd-dll.rc'))]
+    libzstd_sources += [
+      windows_mod.compile_resources(
+        join_paths(zstd_rootdir, 'build/VS2010/libzstd-dll/libzstd-dll.rc'),
+        include_directories: include_directories(join_paths(zstd_rootdir, 'lib')),
```

Comment:
Looking at the output from `meson compile -v -C builddir`, I am not sure `include_directories` are actually passed in to the resource compiler. 

Anyone else observes this issue?

Reply:
It will, but I have MSVC compiler error about syntax: <https://github.com/lzutao/zstd/runs/1264653094#step:8:130>.

Reply:
Interesting, I cannot get beyond compiling resources on my system, whereas in your build it goes passed it and compiles the resource just fine.

That error is not about syntax, it's a linker error, basically it's saying that there were a couple of symbols observed in functions (`ZSTD_cycleLog`, `ZSTD_XXH64`, `ZDICT_trainFromBuffer_unsafe_legacy`) that were not actually defined. Could it be a missing source/header file or compiler/linker flag?

Reply:
That was a shared build. All the missing symbols were in `zstd.lib`, which was included the the `link` command.

There was also a statically build, which has syntax error: <https://github.com/lzutao/zstd/runs/1267569335#step:8:124>.

Reply:
*I am sorry for giving the wrong like*.

Reply:
Oh, so it is, not seen that one